### ISSUE TITLE
Naming of charts and carriers in charts: fix these

### ISSUE THREAD

**Issue by AlexanderWirtz** [quintel/etmodel#1946]
Quite a few carriers and charts have been named confusingly and contain confusing carriers.`

### Final demand mekko needs carrier name update
- Gas should be network gas
- Bio-fuels should be biogenic_fuels

![image](https://cloud.githubusercontent.com/assets/308563/10787126/df8e93d0-7d6e-11e5-8ab8-834f03cc2c35.png)

- `network_gas` entry already exists
- Created new entries for: 

```
biogenic_fuels: "Biogene brandstoffen"
biogenic_fuels: "Biogenic fuels"
```

### Primary demand mekko needs carrier name update
- Biomass should be Solid biomass
- Gas should be Natural gas
- Biofuels should be Liquid biofuels

![image](https://cloud.githubusercontent.com/assets/308563/10787190/29e77f64-7d6f-11e5-9e23-fbad08374793.png)

- `natural_gas` entry already exists
- `bio_fuels` refers to Liquid biofuels
- Created new entries for: 

```
solid_biomass: "Vaste biomassa"
solid_biomass: "Solid biomass"
```

Still need to push this

### Rename Primary demand of biofuels chart
Should be 'Primary demand of biogenic energy carriers' and 'Primair gebruik van biogene brandstoffen' in Dutch

![image](https://cloud.githubusercontent.com/assets/308563/10787265/d97d2bc2-7d6f-11e5-8287-ba4aae0c8e83.png)

I have done this, but need to push this

### Electricity production
- Gas should be Natural gas
- Biomassa should be Solid biomass

![image](https://cloud.githubusercontent.com/assets/308563/10788200/4361d5be-7d76-11e5-84e1-db9a52a962e9.png)

### Import waterfall
- Gas should be Natural gas
- Biomass should be Biogenic ...

![image](https://cloud.githubusercontent.com/assets/308563/10788228/8ac45c4c-7d76-11e5-85d4-9448abd2054f.png)

### Geothermal is confusing in two charts as it is always zero
Residences:
![image](https://cloud.githubusercontent.com/assets/308563/10786716/01af6032-7d6c-11e5-8dcc-4552a86db2cf.png)

Other buildings:
![image](https://cloud.githubusercontent.com/assets/308563/10786723/0d5b2e0c-7d6c-11e5-8922-f98600fea531.png)

### Solar thermal is confusing as it reduces demand and is always zero
This confuses people.
![image](https://cloud.githubusercontent.com/assets/308563/10786915/67bdbe7c-7d6d-11e5-8d22-a38e76ab7546.png)

### Solar thermal does show up, but not the way it does for households
This also confuses people, why is this part of space heating here and not solar panels, like in residences. Why does the model not treat this as reducing heat demand? Guess: because there is no hot water demand in Buildings.
![image](https://cloud.githubusercontent.com/assets/308563/10787004/0927c564-7d6e-11e5-8fee-1aa128ad230b.png)

**Comment by RobTerwel** [comment]
Assigned to @Robterwel

**Comment by RobTerwel** [comment]
Progress overview

- [ ] Final energy demand mekko
  - [x] output elements
  - [ ] interface; To do: `biogenic_fuels` instead of `bio_fuels`
- [ ] Primary energy use mekko
  - [x] output elements
  - [ ] interface; To do: `solid_biomass` instead of `biomass`
- [x] Primary demand of biofuels
- [ ] Electricity production; To do: `solid_biomass` instead of `biomass`
- [ ] Import waterfall present; To do: `biogenic_fuels` instead of `biomass`
- [ ] Import waterfall future; To do: `biogenic_fuels` instead of `biomass`
- [ ] Final energy demand for heating in residences (households)
  - [ ] geothermal
  - [ ] solar thermal
- [ ] Final energy demand for heating in non-residences (buildings) 
  - [ ] geothermal
  - [ ] solar thermal

All interface re-labelling will be done after the pull request for output elements has been merged to prevent charts from not showing up on beta.

@AlexanderWirtz , do I understand you correcting in thinking you would like to see the geothermal output_element_series removed altogether from both final energy demand for heating graphs? 

And as to the solar, the label for residences is `solar` and the label for buildings is `solar_thermal`. Do you mean to say they should be the same or would you like to see some more fundamental changes?

**Comment by AlexanderWirtz** [comment]
I mean it should be solar thermal everywhere it is meant to be solar thermal. However in charts where solar thermal will never show up, it should not be in the legend.

**Comment by RobTerwel** [comment]
Merging closed this issue automatically because of the 'fixes #1946', but the interface elements still have to be changed in some cases (see overview above). I will take care of that now.

**Comment by RobTerwel** [comment]
All charts and carriers listed above have been edited; closing.
